src/recommend.test.js

```diff
@@ -1,34 +1,62 @@
 const { recommend } = require('../');
 
-test('valid single forestType recommendations', () => {
-  expect(recommend('60')).toStrictEqual({
-    one: [302800],
-    three: [800, 25400, 60400, 60500, 227200, 363700],
-    two: [402300],
+describe('Test for input values', () => {
+  test('at least one forest Type required', () => {
+    expect(() => recommend()).toThrowError(
+      `at least one forest type is required to get the recommendation of tree species`,
+    );
+  });
+
+  test('invalid forestType1', () => {
+    expect(() => recommend('963D')).toThrowError(
+      `963D is not a valid forest Type`,
+    );
+  });
+
+  test('invalid forestType2', () => {
+    expect(() => recommend('60', '963D')).toThrowError(
+      `963D is not a valid forest Type`,
+    );
   });
-});
 
-test('valid multiple forestType recommendations', () => {
-  expect(recommend('60', '50')).toStrictEqual({
-    one: [302800, 100],
-    two: [402300, 800],
-    three: [
-      800,
-      25400,
-      60400,
-      60500,
-      227200,
-      363700,
-      25300,
-      174200,
-      317500,
-      402200,
-    ],
+  test('invalid argument type for future flag', () => {
+    expect(() => recommend('60', '963D')).toThrowError(
+      `963D is not a valid forest Type`,
```

Comment:
This seems wrong to me.

Reply:
yeah, thanks for catching it. Now fixed.